hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -54,13 +54,23 @@
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientMapBasicTest {
+    private static final String KEY = "key";
```

Comment:
I just relieved you from that pain:
```
private static final String KEY = randomString();
private static final String OLD_VALUE = randomString();
private static final String NEW_VALUE = randomString();
```
It is completely irrelevant what's the content of those placeholders. They have to be unique, that's the only requirement. It's just the meta info which is relevant for the test case. This is why I consider this as a good practice. It saves you to parse and compare the content of Strings in each test, because this is not relevant for this kind of tests.